mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/BlockMatrix.scala

```diff
@@ -280,17 +278,24 @@ class BlockMatrix @Since("1.3.0") (
     }.groupByKey().map { case (rowIdx, vectors) =>
```

Comment:
At line 276, do we need to call `asBreeze` for `vector`. I think that it is okay with `(blockIndex, vector)` to avoid an object creation in a method `asBreeze`.